config.go

```diff
@@ -378,6 +379,8 @@ type Config struct {
 
 	CoinSelectionStrategy string `long:"coin-selection-strategy" description:"The strategy to use for selecting coins for wallet transactions." choice:"largest" choice:"random"`
 
+	CoinSelectionStrategyObject wallet.CoinSelectionStrategy `description:"The coin selection strategy wallet object to use for selecting coins for wallet transactions." hidden:"true"`
```

Comment:
This value is not required. You can simply access the value through `r.server.cc.Wallet.Cfg.CoinSelectionStrategy` in the RPC server.